modules/test/playwright/tests/portal-impl/main/test.properties

```diff
@@ -0,0 +1 @@
+testray.main.component.name=Portal Services
```

Comment:
Used the same name in the `portal-impl/test.properties` https://github.com/liferay/liferay-portal/blob/0edce38d446831c6b51e06f4f52a1db17e8130a9/portal-impl/test.properties#L14